### DISCUSSION TITLE
Locale Issue on Page Navigation

### DISCUSSION THREAD

**Discussion by Power-tile** [requarks/wiki#3331] [Error / Bug Report]
**Describe the bug**
Changing the page binding for one navigation link in one locale also influences the page binding's locale for the same navigation link in another locale.

**To Reproduce**
1. Change one page binding (e.g. ```/special/test``` which will be mapped to ```/en/special/test```) for one navigation link in the English Locale.
2. Go to the Chinese Locale navigation, and the corresponding page binding for the same navigation link is ```/en/special/test```, not ```/zh/special/test```. Changing this to ```/zh/special/test``` will cause the English page binding to become ```/zh/special/test```.

**Expected behavior**
Changing the page binding in one locale should change the configuration of other locales on the same navigation link to the corresponding locale's page (if exists).

**Host Info (please complete the following information):**
 - OS: Ubuntu 18.04 LTS
 - Wiki.js version: 2.5.170
 - Database engine: MariaDB Ver 15.1; Distribution 10.5.8

**Comment by sarajw** [comment]
I also have this problem. 

![image](https://user-images.githubusercontent.com/88535714/128488486-4e920caf-a21e-4494-8747-d669665fb001.png)

Well, I beg to differ. I was forced to copy the navigation over from my English into my German locale, and yet now I can't edit the new German one without also editing the English one.
